docs/DeveloperGuide.md

```diff
@@ -4,7 +4,10 @@
   pageNav: 3
 ---
 
-# InternBuddy Developer Guide
+<h1>
+    <img src="images/InternBuddyLogo.png" alt="Logo" width="40" height="40" style="vertical-align:middle;">
+    InternBuddy Developer Guide
+</h1>
```

Comment:
Suggestion to change to
```
# InternBuddy Developer Guide<img src="images/InternBuddyLogo.png" alt="Logo" width="40" height="40" style="vertical-align:middle;">
```
instead to align with User Guide, also if back to the top hyperlink is implemented, it may be referenced via
```
[back to top](#internbuddy-developer-guide)
```